ci/deploy.sh

```diff
@@ -87,6 +98,3 @@ ghp-import \
   --branch=gh-pages \
   --message="$MESSAGE" \
   webpage
-
-# Workaround https://github.com/travis-ci/travis-ci/issues/8082
```

Comment:
Was this Travis issue resolved?

Reply:
We put this here in https://github.com/manubot/rootstock/pull/23. I think it's been resolved upstream with Travis, but we kept it in case there was a Travis regression.